modules/library/main/src/main/java/org/geotools/util/CommonsConverterFactory.java

```diff
@@ -220,8 +220,8 @@
     }
     
     static class DateConverter implements Converter {
-        private static SimpleDateFormat format1 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.S a" );
-        private static SimpleDateFormat format2 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ssa" );
+        private SimpleDateFormat format1 = new SimpleDateFormat( "yyyy-MM-dd HH:mm:ss.S a" );
```

Comment:
Moved them into the method, and only create as needed. I still don't like the overhead here as this converter is likely called a fair bit. Lets see how it does on master and we can always do a "quick" regex check before having to create this single threaded SimpleDateFormat.